src/io/flutter/view/FlutterView.java

```diff
@@ -623,12 +627,12 @@ protected void presentClickableLabel(ToolWindow toolWindow, List<LabelInput> lab
 
     for (LabelInput input : labels) {
       if (input.listener == null) {
-        final JLabel descriptionLabel = new JLabel(input.text);
+        final JLabel descriptionLabel = new JLabel("<html>" + input.text + "</html>");
         descriptionLabel.setBorder(JBUI.Borders.empty(5));
         descriptionLabel.setHorizontalAlignment(SwingConstants.CENTER);
         panel.add(descriptionLabel, BorderLayout.NORTH);
       } else {
-        final LinkLabel<String> linkLabel = new LinkLabel<>(input.text, null);
+        final LinkLabel<String> linkLabel = new LinkLabel<>("<html>" + input.text + "</html>", null);
```

Comment:
adding <html> seems odd. Are they needed? this is still far from valid html with <html> tags added.

Reply:
I added this to get text wrapping within the panel. I read that we can also get text wrapping by changing to a text area with some other adjustments, but this seems more straightforward.